social_core/tests/backends/open_id_connect.py

```diff
@@ -177,5 +193,41 @@ def test_invalid_issue_time(self):
     def test_invalid_nonce(self):
         self.authtoken_raised(
             'Token error: Incorrect id_token: nonce',
-            nonce='something-wrong'
+            nonce='something-wrong',
+            kid='testkey',
         )
+
+    def test_invalid_kid(self):
+        self.authtoken_raised('Token error: Signature verification failed', kid='doesnotexist')
+
+
+class ExampleOpenIdConnectAuth(OpenIdConnectAuth):
+    name = 'example123'
+    OIDC_ENDPOINT = 'https://example.com/oidc'
+
+
+class OpenIdConnectTest(OpenIdConnectTestMixin, OAuth2Test):
```

Comment:
Tests are not executed from this file, move it to `test_*.py`, for example  `social_core/tests/backends/test_open_id_connect.py`.

Reply:
Oh, good find! Thanks for bringing this up.

I guess it was confused that pytest actually executed the tests when I was specifying `pytest social_core/tests/backends/open_id_connect.py` directly.

Reply:
Fixed it.

Reply:
The mixin was probably intentionally in a non-matching filename - it does not make sense to execute it standalone. But it does no harm to have it in matching one as it does not inherit from UnitTest class...